Dockerfile

```diff
@@ -46,34 +80,32 @@ RUN dnf config-manager --disable && \
     dnf -y --setopt=tsflags=nodocs install java-11-openjdk wget unzip &> /dev/null && \
     dnf -y clean all
 
-# Prepare Management Center
-RUN wget -O ${MC_HOME}/${MC_INSTALL_ZIP} \
-          http://download.hazelcast.com/management-center/${MC_INSTALL_ZIP} \
- && unzip ${MC_INSTALL_ZIP} \
-      -x ${MC_INSTALL_NAME}/docs/* \
- && rm -rf ${MC_INSTALL_ZIP} \
- && mv ${MC_INSTALL_NAME}/* . \
- && rm -rf ${MC_INSTALL_NAME}
+RUN echo "Installing new packages" \
+    && dnf -y --setopt=tsflags=nodocs install java-11-openjdk &> /dev/null \
```

Comment:
Why is this `dnf` repeated here again. I guess you already executed that in the line 80.

Reply:
removed, thanks. leftover from c/p